ietf-network-inventory.yang

```diff
@@ -2,12 +2,6 @@ module ietf-network-inventory {
   yang-version 1.1;
   namespace "urn:ietf:params:xml:ns:yang:ietf-network-inventory";
   prefix ni;
-
-  import iana-hardware {
-    prefix ianahw;
-    reference
-      "https://www.iana.org/assignments/yang-parameters";
-  }
```

Comment:
I think you need to import iana-hardware: it is used by the class data node below

Reply:
Thank you, it is added back.